server/app/routes/plots/plots.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+// assume all routes begin with /api/plots
+var router = require('express').Router();
+var _ = require('lodash');
+var db = require('../../../db/_db.js');
+var Plot = db.model('plot');
+var PlotPlants = db.model('PlotPlants');
+
+// get all plots
+router.get('/', function(req, res, next) {
+	Plot.findAll()
+	.then(function(plots) {
+		res.send(plots);
+	})
+	.catch(next);
+});
+
+// get plot by id
+router.get('/:id', function(req, res, next) {
+	Plot.findById(req.params.id)
+	.then(function(plot) {
+		res.send(plot);
+	})
+	.catch(next);
+});
+
+// creates a plot associated with a specific user (aliased as Gardener) -- pass in 
+// userId as part of req.body!!
+router.post('/', function(req, res, next) {
+	Plot.create(req.body)
+	.then(function(plot) {
+		plot.setUser(req.body.userId);
+		res.send(plot);
+	})
+	.catch(next);
+});
+
+// updating plot by adding a plant
+router.put('/:plotId/:plantId', function(req, res, next) {
```

Comment:
In typical REST architecture, this would be `/:plotId/plants/:plantId`. @Dielentheis